notebooks/en/enterprise_cookbook_gradio.ipynb

```diff
@@ -0,0 +1,5796 @@
+{
```

Comment:
<p>transformers under the hood has huggingface_hub as a dependency so I don't think we need to install</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>